contracts/extensions/OneTxPayment.sol

```diff
@@ -24,6 +24,8 @@ import "./ColonyExtension.sol";
 
 
 contract OneTxPayment is ColonyExtension {
+  event OneTxPaymentMade();
```

Comment:
No arguments?

Reply:
What arguments would you expect here? Everything relevant is already emitted in events during the transfer, I believe. This is just indicating that the extension was used for the payment rather than it being 'native'.

Reply:
Yeah that's true actually -- funding, expenditure, etc already do their events.